src/view/components/toolbar-add-contentlet/toolbar-add-contentlet-body.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../scss/mixins";
+@import "../../../scss/variables";
+
+:host {
+
+  .toolbar-add-contentlet-body__title{
```

Comment:
I dont think the name should have ```add-contentlet``` because we can also add widgets, pages, etc...